litestar/app.py

```diff
@@ -466,18 +468,24 @@ def serialization_plugins(self) -> list[SerializationPluginProtocol]:
         return list(self.plugins.serialization)
 
     @staticmethod
-    def _get_default_plugins() -> list[PluginProtocol]:
-        default_plugins: list[PluginProtocol] = []
+    def _get_default_plugins(plugins: list[PluginProtocol] | None = None) -> list[PluginProtocol]:
```

Comment:
Is there a better place to do this it seems like we would keep adding things to it and this would become quite long.. or just it just stop at `attrs` and `pydantic` plugs?

Reply:
I think we should refactor this if we add more than these, but, at the moment, it's just these two.